sdk/storage/azblob/zt_examples_test.go

```diff
@@ -124,15 +118,325 @@ func Example() {
 	}
 
 	// Delete the blob.
-	_, err = blockBlobClient.Delete(context.TODO(), nil)
+	_, err = client.DeleteBlob(context.TODO(), containerName, blobName, nil)
 	handleError(err)
 
 	// Delete the container.
-	_, err = containerClient.Delete(context.TODO(), nil)
+	_, err = client.DeleteContainer(context.TODO(), containerName, nil)
 	handleError(err)
 }
 
-// ---------------------------------------------------------------------------------------------------------------------
+// tiny blob client ----------------------------------------------------------------------------------------------------
```

Comment:
```suggestion
// blob client ----------------------------------------------------------------------------------------------------
```

Reply:
`blob client` is different from `tiny blob client`. It basically represents the collection of APIs exposed to general user (not super user).

Reply:
There is no such thing as "tiny blob" to a customer; this is just a term we've been using internally.  So the term shouldn't be leaked exteranlly.

Reply:
Please not that this will eventually go one step further.  i.e. the _only_ examples we should have in the `azblob` package are for its contents.  Everything else will be moved to the corresponding sub packages.